### DISCUSSION TITLE
西格玛文件管理器 v1.7打不开

### DISCUSSION THREAD

**Discussion by dangzw** [aleksey-hoffman/sigma-file-manager#309] [Q&A]
一直卡在欢迎界面
![微信截图_20231226111936](https://github.com/aleksey-hoffman/sigma-file-manager/assets/20716929/7f2f5880-f6e0-4e0a-b086-9e081a537efa)

**Comment by dangzw** [comment]
win10系统

**Comment by aleksey-hoffman** [comment]
Hi, thanks for the report. This problem will be fixed in [versoin 2.0](https://github.com/aleksey-hoffman/sigma-file-manager/discussions/226) later. Unfortunatelly I don't have time to fix this in the current version

**Comment by dangzw** [comment]
好的，感谢回复，期待更好的作品。












D568509132
***@***.***


---原始邮件---
发件人: "Aleksey ***@***.***&gt;
发送时间: 2023年12月26日(周二) 下午5:56
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [aleksey-hoffman/sigma-file-manager] 西格玛文件管理器 v1.7打不开 (Discussion #309)




 
Hi, thanks for the report. This problem will be fixed in versoin 2.0 later. Unfortunatelly I don't have time to fix this in the current version
